src/web/client/context/entityDataMap.ts

```diff
@@ -49,7 +50,8 @@ export class EntityDataMap {
             entityName,
             odataEtag,
             entityColumnMap,
-            mappingEntityId
+            mappingEntityId,
+            filePath
```

Comment:
this will end up over-writing any previosuly saved filepaths for an entity like webpage